analysis/backtrace/backtrace.go

```diff
@@ -139,6 +139,18 @@ func Analyze(logger *config.LogGroup, cfg *config.Config, prog *ssa.Program, pkg
 	var errs []error
 	resTraces := make(map[df.GraphNode][]Trace)
 	for _, ps := range cfg.SlicingProblems {
+		// Number of alarms is problem specific, not global
+		state.ResetAlarms()
+
+		state.Logger.Infof("Analyzing slicing problem %s", ps.Tag)
+		// Set problem-specific options
+		prevOptions := state.Config.AnalysisProblemOptions
+
+		// Overriding options with problem-specific config
+		if ps.AnalysisProblemOptions != nil {
+			config.OverrideWithAnalysisOptions(state.Logger, state.Config, ps.AnalysisProblemOptions)
```

Comment:
We should log a message when overriding an option, preferably at info level

Reply:
Yes, the function `OverrideWithAnalysisOptions` does log using the `state.Logger`.